README.md

```diff
@@ -1,28 +1,15 @@
-== README
+# Schedulr
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Description
 
-Things you may want to cover:
 
-* Ruby version
+## Steps to get working locally
 
-* System dependencies
+  - Clone
+  - Bundle Install
+  - Run `rake db:setup`
```

Comment:
This will cause a problem if the person who cloned the app, did so before there were any migrations.